spec/lib/active_data/model/dirty_spec.rb

```diff
@@ -74,7 +74,7 @@
   specify { expect(Model.new(title: 'Hello').changes).to eq('title' => [{}, 'Hello']) }
   specify { expect(Model.new(title_translations: {en: 'Hello'}).changes).to eq('title' => [{}, 'Hello']) }
 
-  specify { expect(Model.new).not_to respond_to :something_changed? }
+  specify { expect(Model.new).to respond_to :something_changed? }
```

Comment:
This is a questionable change.

Reply:
something is an embeds_one association, there should not be dirty methods for associations.